aziotctl/src/system.rs

```diff
@@ -38,12 +41,24 @@ pub struct LogLevelOptions {
     log_level: log::Level,
 }
 
-pub fn system(options: Options) -> Result<()> {
+#[derive(StructOpt)]
+#[structopt(about = "Reprovision device with IoT Hub")]
+pub struct ReprovisionOptions {
+    #[structopt(
+        value_name = "Identity Service URI",
+        long = "--uri",
```

Comment:
```diff,rust
-long = "--uri",
+long = "uri",
```

Reply:
Actually, this shouldn't even be a parameter. The user doesn't get to override the endpoints anyway.

Reply:
True, but it's useful for development. For example, on my test VM I'm not using the default endpoints during development. This lets me use the reprovision command on my dev endpoints.

Reply:
My point is it shouldn't be a parameter of the CLI that the user can set.

Either `#[cfg(debug_assertions)]` it, or read it from an env var.

Reply:
Fixed